.travis.yml

```diff
@@ -88,13 +88,13 @@ jobs:
     sudo: required
     services:
       - docker
-    env: CC=clang-7 CXX=clang++-7 TARGET=STATICANALYSIS ECO="-DUSE_CLANG_TIDY=ON -DWITH_OPENMP=OFF"
```

Comment:
I'm very sure this should stay as-is :)
There is no libomp in docker image https://hub.docker.com/r/darktable/rawspeed/builds/bkscbkihgtmqags6bakwylb/

Reply:
It's an error if OpenMP is not detected in the system?
I will remove this commit.

Reply:
> It's an error if OpenMP is not detected in the system?

It is since 2a3b9dc913d2cb95965b7e545044cd23a4e04293

> I will remove this commit.

Did that when merging.